slices/x11-common.yaml

```diff
@@ -0,0 +1,30 @@
+package: x11-common
+
+essential:
+  - x11-common_copyright
+
+slices:
+  data:
+    essential:
+      - lsb-base_bins
```

Comment:
I don't think this would be needed here. What do you need? `lsb-base` just has copyright stuff (so you if that's all you need, it should be a dependency of the `copyright` slice).

Then `lsb-base` brings, by dependency, sysvinit-utils, which I wonder if are only used for this package's `postinst` script? If that's the case, then this comment is very similar to the above, in the sense that you're either missing a mutation scripts, or you have an unnecessary dependency for this use case/slice

Reply:
The package itself does depend on lsb-base. In reality we don't start the process and we don't need sysvinit-utils for it to run. I will remove the slice for this usecase (because we're slicing for the library at the end of the day).